### DISCUSSION TITLE
New release when?

### DISCUSSION THREAD

**Discussion by christher-lenander** [Stun3R/strapi-sdk-js#176] [General]
I saw a few closed tickets with upload and change password. When will the next release come out?

**Comment by Stun3R** [comment]
The new release should come out by the end of the week. I'm cleaning some stuff and test many things as the build changed.

**Comment by Stun3R** [comment]
The new version has been released: `v2.3.0`. Check [here](https://github.com/Stun3R/strapi-sdk-js/releases/tag/v2.3.0) for more details
